sql/select.go

```diff
@@ -204,9 +204,10 @@ func (p *planner) selectIndex(s *scanNode, ordering []int) (planNode, error) {
 	sort.Sort(indexInfoByCost(candidates))
 
 	if log.V(2) {
+		// TODO(pmattis): Show all of the constraints?
```

Comment:
I added display of the constraints in this PR. The format is reasonably compact for the moment.